common4j/src/main/com/microsoft/identity/common/java/authorities/AzureActiveDirectoryAudience.java

```diff
@@ -154,6 +164,32 @@ public void setTenantId(String tenantId) {
         mTenantId = tenantId;
     }
 
+    /**
+     * Determines the audience type (MSA, Common, AAD, or Unknown) from the given
+     * {@link AzureActiveDirectoryAuthority}.
+     *
+     * @param authority The authority from which the audience is derived.
+     * @return The audience type: "MSA", "Common", "AAD", or "Unknown".
+     */
+    public static AudienceTelemetryConstant getAudienceFromAuthority(final AzureActiveDirectoryAuthority authority) {
```

Comment:
```suggestion
    public static AudienceTelemetryConstant getAudienceForTelemetryFromAuthority(final AzureActiveDirectoryAuthority authority) {
```

Reply:
Slight suggesttion above for method rename

Reply:
added